case-lib/lib.sh

```diff
@@ -339,6 +339,33 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+# Prints the syst collateal file found on stdout, logs on stderr.
+find_syst_dict_file()
+{
+    local dict_file
+    # if user doesn't specify file path of sof-*.ldc, fall back to
+    # /etc/sof/sof-PLATFORM.ldc, which is the default path used by CI.
+    # and then on the standard location.
+    if [ -n "$SOFSYST" ]; then
+        dict_file="$SOFSYST"
+        >&2 dlogi "SOFSYST=${SOFSYST} overriding default locations"
+    else
```

Comment:
`printf` and early `return` here avoids the `else` and the extra indentation and is easier to follow. Don't even need the `dict_file` variable yet.